src/libs/OptionsListUtils.js

```diff
@@ -204,20 +205,58 @@ function hasReportDraftComment(report) {
         && lodashGet(reportsWithDraft, `${ONYXKEYS.COLLECTION.REPORTS_WITH_DRAFT}${report.reportID}`, false);
 }
 
+/**
+ * @param {Object} report
+ * @returns {Array<String>}
+ */
+function getParticipants(report) {
+    const participants = [...lodashGet(report, ['participants'], [])];
+    if (ReportUtils.isPolicyExpenseChat(report) && !_.contains(participants, report.ownerEmail)) {
+        participants.push(report.ownerEmail);
+    }
+    return participants;
+}
+
+/**
+ * Get the Avatar urls or return the icon according to the chat type
+ *
+ * @param {Object} report
+ * @returns {Array<*>}
+ */
+function getAvatarSources(report) {
+    return _.map(lodashGet(report, 'icons', ['']), (source) => {
+        if (source) {
+            return source;
+        }
+        if (ReportUtils.isArchivedRoom(report)) {
+            return Expensicons.DeletedRoomAvatar;
+        }
+        if (ReportUtils.isChatRoom(report)) {
+            return Expensicons.ActiveRoomAvatar;
+        }
+        if (ReportUtils.isPolicyExpenseChat(report)) {
+            return Expensicons.Workspace;
+        }
+        return Expensicons.Profile;
```

Comment:
I added `Expensicons.Profile` to remove `[personalDetail.avatar]` from `createOption` function: 

https://github.com/Expensify/App/blob/00805b2b495fac25f53a42231dd3e5ae78138a2a/src/libs/OptionsListUtils.js#L259

I realized that `personalDetail.avatar` is not the correct solution when the icon is still not present (which happens when the app is bootstrapping and loading the reports for the first time) because `personalDetail` is the first option of the array `personalDetails` and the first option not always correspond to the real report avatar (in some cases it shows for a second the avatar of the user logged in and then the real avatar).